test_psa_target.py

```diff
@@ -246,7 +262,7 @@ def _get_parser():
         "--mcu",
         help="Build for the given MCU",
         required=True,
-        choices=["ARM_MUSCA_B1"],
+        choices=["ARM_MUSCA_B1", "ARM_MUSCA_S1"],
```

Comment:
Please use `_get_tfm_regression_targets()` https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/build_tfm.py#L104 by defining in `psa_builder.py` in its own commit. So that the targets are shared across all the scripts.

Reply:
I defined `_get_tfm_regression_targets()` in `psa_builder.py`, although it also worked being defined in `build_tfm.py`.

Reply:
I think we should keep in `psa_builder.py` as this is more generic.